partials/footer.hbs

```diff
@@ -1,32 +1,47 @@
-<footer class="site-footer container">
-    <div class="social">
-        <a class="social-item social-item-facebook"
-            href="https://www.facebook.com/advisorysg/" target="_blank"
-            rel="noopener noreferrer" aria-label="Facebook" >
-            <svg class="icon">
-                <use xlink:href="#facebook-box"></use>
-            </svg>
-        </a>
-        <a class="social-item social-item-instagram"
-            href="https://www.instagram.com/advisory.sg/" target="_blank"
-            rel="noopener noreferrer" aria-label="Instagram">
-            <svg class="icon">
-                <use xlink:href="#instagram-box"></use>
-            </svg>
-        </a>
-        <a class="social-item social-item-linkedin"
-            href="https://www.linkedin.com/company/advisorysg/" target="_blank"
-            rel="noopener noreferrer" aria-label="LinkedIn">
-            <svg class="icon">
-                <use xlink:href="#linkedin-box"></use>
-            </svg>
-        </a>
+<footer class="flex flex-col align-center">
```

Comment:
Please retain the classes `site-footer container` if possible! Do modify the relevant classes in `assets/css` if necessary.